### PR TITLE
feat: frontend features

### PR DESCRIPTION
**Pull request by dtscalac**
# Description

Merges feature branch where all flutter related changes were done during earthly downtime (due to cache).

The code has been already reviewed in all PRs that landed in this branch.

### CODE REVIEW

**Review by minikin — APPROVED** [review]
approved with the minor suggestion.

File: catalyst_voices/lib/pages/widgets/voices_app_bar/actions/finish_account_button.dart [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+class FinishAccountButton extends StatelessWidget {
+  final VoidCallback? onPressed;
+
+  const FinishAccountButton({
+    super.key,
+    this.onPressed,
+  });
+
+  @override
+  Widget build(BuildContext context) {
```
Comment (minikin) [inline catalyst_voices/lib/pages/widgets/voices_app_bar/actions/finish_account_button.dart:15]: Don't want to be picky but refactor to widget, please

File: catalyst_voices/lib/pages/widgets/voices_app_bar/actions/notifications_indicator.dart [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  Widget build(BuildContext context) {
+    return badgeText != null
+        ? Badge(
+            label: Text(badgeText!),
+            child: _iconButton,
+          )
+        : _iconButton;
+  }
+
+  Widget get _iconButton => IconButton.outlined(
```
Comment (minikin) [inline catalyst_voices/lib/pages/widgets/voices_app_bar/actions/notifications_indicator.dart:26]: Don't want to be picky but refactor to widget, please

Comment (dtscalac) [reply catalyst_voices/lib/pages/widgets/voices_app_bar/actions/notifications_indicator.dart:26]: Alright, will do in the next PR 👍

**Pull request merged**